Documentation/cmdref/cilium-agent.md

```diff
@@ -1,349 +0,0 @@
-<!-- This file was autogenerated via cilium-agent --cmdref, do not edit manually-->
```

Comment:
Something is wrong here. This commit removes all `cmdref` documentation, instead of regenerating it. I'd recommend dropping this commit and doing the `cmdref` change in the commit which also removes the flag in `daemon_main.go`.